logback-classic/src/main/java/ch/qos/logback/classic/encoder/JsonEncoder.java

```diff
@@ -264,40 +268,66 @@ private void appendThrowableProxy(StringBuilder sb, String attributeName, IThrow
         }
 
         sb.append(VALUE_SEPARATOR);
-        appendSTEPArray(sb, itp.getStackTraceElementProxyArray(), itp.getCommonFrames());
+        if (withOriginalStackTrace) {
+            appenderMember(sb, "stackTrace", getOriginalStackTrace(itp));
```

Comment:
shouldn't `"stackTrace"` be a constant like `THROWABLE_ATTR_NAME` is a constant?